sdk/core/Azure.Core/api/Azure.Core.net5.0.cs

```diff
@@ -544,8 +544,11 @@ public abstract partial class RequestContent : System.IDisposable
         public static Azure.Core.RequestContent Create(byte[] bytes) { throw null; }
         public static Azure.Core.RequestContent Create(byte[] bytes, int index, int length) { throw null; }
         public static Azure.Core.RequestContent Create(System.IO.Stream stream) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("This method uses reflection-based serialization which is incompatible with trimming. Try using one of the 'Create' overloads that doesn't wrap a serialized version of an object.")]
```

Comment:
Curious: I think you have told me before, but these are only in the net5+ TFMs because trimming didn't exist before that version?

Reply:
Also (more curiosity): why do these only show up on RequestContent APIs and not APIs for other types?

Reply:
This PR only annotates the parts of the Azure.Core which are needed by the `Azure.Monitor.OpenTelemetry.Exporter`. It's not trying to make the whole of Azure.Core trim/AOT compatible.

Reply:
Thank you, that makes sense!  I guess I'm also wondering why the AOT-related attributes don't show up for other public types like DynamicData and HttpPipelineSynchronousPolicy?

Reply:
Oh nevermind, I just missed them in the file.  🤦🏻‍♀️